devops/test_configs.json

```
@@ -2,22 +2,22 @@
   "lts": [
     {
       "config": "lin_intel",
-      "name": "SYCL E2E on Intel CPU/GEN9 GPU",
+      "name": "SYCL E2E on Intel CPU/GEN12 GPU",
       "runs-on": [
         "Linux",
-        "gen9"
+        "gen12"
       ],
       "image": "${{ inputs.intel_drivers_image }}",
       "container_options": "-u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN",
       "targets": "ext_oneapi_level_zero:gpu;opencl:gpu;opencl:cpu",
       "cmake_args": ""
     },
     {
-      "config": "l0_gen9",
-      "name": "L0 GEN9 LLVM Test Suite",
+      "config": "l0_gen12",
```

I think we can drop the device architecture name here. L0 doesn't support other device architectures. I think we copy-pasted the OpenCL config, which supports multiple device architectures.
```suggestion
      "config": "l0",
```

---

Well, it can be "l0_pvc", etc. Anyway, dropping should come in its own PR.